### PR TITLE
Open selection mark list item as read.

### PR DESCRIPTION
**Pull request by vanniktech**
Other part of #1240

I tested it locally and it works. Forgive me if this is not the way to do it - I'm no JS developer.

### CODE REVIEW

File: source/features/add-selection-in-new-tab.js [hunk 1]
```diff
@@ -11,6 +11,11 @@ export default function () {
 				urls: [selected.href],
 				action: 'openAllInTabs'
 			});
+
+			// Get the list element that contains the unread class and mark it as read.
+			const classList = selected.parentElement.parentElement.classList;
```
Comment (vanniktech) [inline source/features/add-selection-in-new-tab.js]: Open to a different way to jump from the `<a>` back to the `<li>`.

**Comment by vanniktech** [comment]
CI somehow built an old commit but tagged the new one with a failing status check. Reopening.

File: source/features/add-selection-in-new-tab.js (hunk 1 above)
Comment (bfred-it) [reply source/features/add-selection-in-new-tab.js]: `.closest()`

File: source/features/add-selection-in-new-tab.js [hunk 2]
```diff
@@ -11,6 +11,11 @@ export default function () {
...
+			classList.remove('unread');
+			classList.add('read');
```
Comment (bfred-it) [inline source/features/add-selection-in-new-tab.js]: Merge these three lines by using `classList.replace()`

**Comment by vanniktech** [comment]
@bfred-it friendly ping

**Comment by vanniktech** [comment]
Can you take another look? I've addressed your feedback.

**Comment by bfred-it** [comment]
Thanks!

**Pull request merged**